openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,579 @@
+
+  const hasEditPermission = useMemo(() => !tagItem?.disabled, [tagItem]);
+
+  const onDescriptionUpdate = async (updatedHTML?: string) => {
+    if (tagItem) {
+      if (tagItem.description !== updatedHTML) {
+        const updatedTableDetails = {
+          ...tagItem,
+          description: updatedHTML,
+        };
+        const jsonPatch = compare(tagItem, updatedTableDetails);
+        try {
+          const response = await patchTag(tagItem.id ?? '', jsonPatch);
+
+          setTagItem(response);
+
+          if (tagItem?.name !== updatedTableDetails.name) {
+            history.push(
+              getClassificationTagPath(response.fullyQualifiedName ?? '')
+            );
+          }
+        } catch (error) {
+          showErrorToast(error as AxiosError);
+        }
+        setIsDescriptionEditable(false);
+      } else {
+        setIsDescriptionEditable(false);
+      }
+    }
+  };
+
+  const getTagData = async () => {
+    try {
+      setIsLoading(true);
+      if (tagFqn) {
+        const response = await getTagByFqn(tagFqn);
+        setTagItem(response);
+      }
+    } catch (e) {
+      if (e instanceof AxiosError) {
+        showErrorToast(e);
+      }
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const activeTabHandler = (tab: string) => {
+    if (tagItem) {
+      history.push({
+        pathname: getClassificationTagPath(
+          tagItem.fullyQualifiedName ?? '',
+          tab
+        ),
+      });
+    }
+  };
+
+  const editDescriptionPermission = useMemo(() => {
+    const editDescription = checkPermission(
+      Operation.EditDescription,
+      resourceType,
+      permissions
+    );
+
+    return editDescription;
+  }, [permissions, resourceType]);
+
+  const updateTag = async (updatedData: Tag) => {
+    if (tagItem) {
+      const jsonPatch = compare(tagItem, updatedData);
+
+      try {
+        const response = await patchTag(tagItem.id ?? '', jsonPatch);
+
+        setTagItem(response);
+
+        if (tagItem?.name !== updatedData.name) {
+          history.push(
+            getClassificationTagPath(response.fullyQualifiedName ?? '')
+          );
+        }
+      } catch (error) {
+        showErrorToast(error as AxiosError);
+      }
+    }
+  };
+
+  const onNameSave = async (obj: { name: string; displayName: string }) => {
+    if (tagItem) {
+      const { name, displayName } = obj;
+      let updatedDetails = cloneDeep(tagItem);
+
+      updatedDetails = {
+        ...tagItem,
+        name: name?.trim() || tagItem.name,
+        displayName: displayName?.trim(),
+      };
+
+      await updateTag(updatedDetails);
+      setIsNameEditing(false);
+    }
+  };
+
+  const onStyleSave = async (data: Style) => {
+    if (tagItem) {
+      const style: Style = {
+        color: data.color || undefined,
+        iconURL: data.iconURL || undefined,
+      };
+
+      const updatedDetails = {
+        ...tagItem,
+        style,
+        description: tagItem.description ?? '',
```

Comment:
in style why you have description in the object

Reply:
That was a mismatch I think. I have updated that